test/test_fori_loop_with_while_loop_simple_add_dispatch_in_torch.py

```diff
@@ -12,25 +12,27 @@
 
 
 def _fake_while_loop(cond_fn, body_fn, operands):
-  while cond_fn(*operands):
-    operands = body_fn(*operands)
+  while cond_fn(operands):
+    operands = body_fn(operands)
   return operands
 
 
 class WhileLoopTest(unittest.TestCase):
 
   def test_while_loop_tpu(self):
 
+    device = xm.xla_device()
+
     def cond_fn(x):
-      return x.sum() <= 10
+      ten = torch.ones(1, dtype=torch.int32, device=device)
```

Comment:
cosmetic nit: `ten` can be interpreted as number 10 - suggesting a better name plz

Reply:
Thanks, updated